### ISSUE TITLE
Wiimotes won't return any Gyro values to FreePIE

### ISSUE THREAD

**Issue by meduzo** [ajayyy/Wiimote-FreePIE-VR-Controls#1]
Tested with a Win 8.1 Computer

**Comment by ajayyy** [comment]
Sry, can you put this on the FreePIE page? Thanks

**Comment by meduzo** [comment]
Done :) sorry I misunderstood

**Comment by ajayyy** [comment]
This could be because FreePIE needs Wii motion plus

**Comment by migero** [comment]
works fine on my side with motion plus  

diagnostics.watch(wiimote[i].ahrs.pitch)
diagnostics.watch(wiimote[i].ahrs.roll)
diagnostics.watch(wiimote[i].ahrs.yaw)

**Comment by ajayyy** [comment]
Ah, I see. Sad for me because I don't have wii motion plus. I'll add that to the notice. Thanks!

**Comment by migero** [comment]
do you happen to have LeapMotionFreePIEPlugin i can't find it anywhere and on forum it's down

**Comment by ajayyy** [comment]
Yes I do, I have uploaded it to the github repo. It's so sad that dropbox has broken all of these links.

**Comment by migero** [comment]
tnx a ton

**Comment by jensuss** [comment]
i've been following this github for months, never gotten it to work tho. (also i got motion plus)

**Comment by ajayyy** [comment]
@migero No problem, please tell me if everything works out in the end.

@jensuss Darn, I thought that might be the issue. Have you tried this?

```
diagnostics.watch(wiimote[i].ahrs.pitch)
diagnostics.watch(wiimote[i].ahrs.roll)
diagnostics.watch(wiimote[i].ahrs.yaw)
```

Because someone fixed another issue where the controllers did not pop up at all a couple weeks ago, maybe that was just the issue. Not sure though because I don't have a motion plus controller to test with sadly.

**Comment by migero** [comment]
also adding 
```
i=0

wiimote[i].enable(WiimoteCapabilities.MotionPlus)

```
makes the gyro start working 

but the values are to big need to get divided alot but some progres is on

**Comment by ajayyy** [comment]
Oh, I see. That's how you get it working.

**Comment by ajayyy** [comment]
@migero Hey, how exactly did you get it to work like that? In #4 I am trying to help someone get it working

**Comment by GranServer96** [comment]
Hello! I have a question, how can I only use the gyroscope of the Wiimote
I explain:
I see that all the codes and posts talk about how to use MotionPlus but I only want to use the Wiimote Gyroscope without connecting MotionPlus

Before I could do it in GlovePie using this simple code:

```
PPJoy.Analog3 = -( Wiimote.RawAccX )
PPJoy.Analog4 = -( Wiimote.RawAccZ )
```

But now that I use FreePie and the programming is Python I can't find how to do the same.
